static/src/javascripts/projects/commercial/modules/dfp/dfp-env.js

```diff
@@ -63,4 +63,4 @@ const dfpEnv: DfpEnv = {
     },
 };
 
-export default dfpEnv;
+export { dfpEnv };
```

Comment:
Ah as discussed, not going to do this as it will mean that each property needs to be imported separately (assuming we aren't using export default, which we aren't.)